packages/components/namespace-store/namespace-store-form.tsx

```diff
@@ -135,7 +145,7 @@ const NamespaceStoreForm: React.FC<NamespaceStoreFormProps> = withHandlePromise<
         [PROVIDERS_NOOBAA_MAP[provider]]: {
           [BUCKET_LABEL_NOOBAA_MAP[provider]]: providerDataState.target,
           secret: {
-            name: secretName,
+            name: getName(secretName as K8sResourceCommon),
```

Comment:
why ??
`secretName` is string right ? why are we passing it to `getName` ?

Reply:
it's not string, the K8 resource